### DISCUSSION TITLE
Error log on startup

### DISCUSSION THREAD

**Discussion by mutti-wilson** [GrandOrgue/grandorgue#1109] [Help]
I'm getting some errors I never used to when starting up. I'm on the latest version. Any idea where these entries are that need removed?

![PhotoPictureResizer_20220415_180722_copy_1024x771.jpg](https://user-images.githubusercontent.com/92544601/163655935-edaf6af8-9607-48b4-9f09-3e5512000b9e.jpg)

**Comment by oleg68** [comment]
@mutti-wilson could you post here your preset file (it is stored in the <UserHome>/GrandOrgue/Data directory unless you have redefined it in Program Settings)?

**Reply by mutti-wilson** [reply]
I believe this is the file. It's the most recent file in the directory. The other 2 are 12kb and 11kb and this one is 38kb. Not sure what all is stored in this file. I presume it saves settings for all the organs I have?

**Reply by thbeckpr** [reply]
I'm getting these errors on every sample set I open as well.

**Reply by mutti-wilson** [reply]
I must have forgotten to include the file. Here is is.
[E09755E9918413B5BB438757CC6E7280EEEAE48F-0.zip](https://github.com/GrandOrgue/grandorgue/files/8502939/E09755E9918413B5BB438757CC6E7280EEEAE48F-0.zip)

**Reply by kerkovits** [reply]
I confirm this bug. I have this window since version 3.6.4 for all sample sets.

**Reply by sandstone42** [reply]
I ended up deleting all my cmb files.

**Reply by mutti-wilson** [reply]
A simple re-save fixed the issue. Back to playing.

**Comment by willeke1234** [comment]
Just an idea: did you try the extra panel [here](https://github.com/GrandOrgue/grandorgue/discussions/917#discussioncomment-1878518), Save and did you restore the original ODF?

**Reply by paolo2504** [reply]
willeke , where to find the extra panel please ?

**Reply by willeke1234** [reply]
Click on the link, "View more" and "MonFriesach.zip".

**Reply by mutti-wilson** [reply]
Standard panel. I only use the main panel for the organ. No other windows.

**Reply by eturpault** [reply]
I got the same warning messages with this ODF MonFriesach with the last two SW releases of GrandOrgue. By simply doing a settings save (Ctrl + S) the messages are not appearing anymore.

**Reply by mutti-wilson** [reply]
I will try this tonight and see if that fixes the issue.

**Reply by willeke1234** [reply]
It is a bug and has nothing to do with the MonFriesach ODF. The position of the main window wasn't saved. This is fixed but the fix is not compatible with the existing saved presets. Saving again will remove the warnings.
